chapters/annotations.tex

```diff
@@ -8,6 +8,12 @@ \chapter{Annotations}\label{annotations}
 (For replaceable class declarations with a \lstinline[language=grammar]!constraining-clause! also refer to \cref{constraining-clause-annotations}.)
 The specification in this document defines the semantic meaning if a tool implements any of these annotations.
 
+\section{How to Interpret Annotation Definitions}\label{how-to-interpret-annotation-definitions}
+If the annotation itself is described by a \lstinline!record! \emph{class} the annotation can be seen as a modifier for a \lstinline!record! \emph{component} with the same name.
+A default value for a record member will be used if there is no modifier for that record member.
+If the annotation is declared as a parameter the corresponding modifier must be an evaluable parameter.
+If the annotation is declared as a constant the corresponding modifier must be a literal value.
```

Comment:
In my opinion there is already too much confusion about constant vs literal.  I'm afraid that this would set the confusion in stone.
My suggestion, I believe would be a good topic for a phone meeting discussion, is the following:
```suggestion
If the annotation is declared as \lstinline!parameter! the corresponding modifier must be an evaluable parameter.
If the annotation is declared as \lstinline!$\mathit{literal}$ constant! the corresponding modifier must be a literal value.
```

For a constructive discussion, here are some alternatives I'd like to discuss for literal values, including the current current state of the PR:
- `constant String info …;` (current state of PR)
- _literal_ `constant String info …;` (suggestion above)
- `/*literal*/ constant String info …;`
- _literal_ ` String info …;`

Reply:
Please consider these suggestions so that we can have a vote and move forward:

> * `constant String info …;` (current state of PR)
> * _literal_ `constant String info …;` (suggestion above)
> * `/*literal*/ constant String info …;`
> * _literal_ ` String info …;`

I assume the intent is that  *evaluable* should replace *literal* for parameters.

Reply:
> Please consider these suggestions so that we can have a vote and move forward:
> 
> > * `constant String info …;` (current state of PR)
> > * _literal_ `constant String info …;` (suggestion above)
> > * `/*literal*/ constant String info …;`

Language group:

Poll:
1. Elena, Gerd, Stephan
2. Henrik
3. Henrik, Elena, Gerd, Stephan
Abstain: Quentin, Daeoh, Markus

Conclusion: 3.

Reply:
This has now been implemented.  Resolving.